slices/python3-pyftpdlib.yaml

```diff
@@ -0,0 +1,15 @@
+package: python3-pyftpdlib
```

Comment:
I'm guessing we'll get a request to test this standalone (not just as part of the git tests). Would you like me to write a test on my branch?

Reply:
Sure! Thank you.

Reply:
[Here ya go!](https://github.com/canonical/chisel-releases/blob/5c73271a1027f3f01bcc55f4bc27fd706638565c/tests/spread/integration/python3-pyftpdlib/task.yaml)